### DISCUSSION TITLE
No funciona la licencia.

### DISCUSSION THREAD

**Discussion by neido6** [aclap-dev/video-downloadhelper#357] [Help and Q&A]
### Your question

Como puedo encontrar una llave de licencia.

### Addon & CoApp version and details

Estoy intentando descargar un video de Youtube, me pide licencia y cuando voy a generar una nueva llave me dice lo siguiente "Sorry, we could not find a VDH/Firefox license with associated email xxxxx@gmail.com. Please try with an alternative email address you may have used when purchasing the license." cuando hice una compra de la licencia de por vida.

Deberían de mejorar la forma de gestionar la licencia y no volver un verdadero calvario el encontrar una solución.

**Comment by jcfrog** [comment]
If you have a license problem please email us at licenses@downloadhelper.net
